foundationdb/assets/service_checks.json

```diff
@@ -6,6 +6,6 @@
     "statuses": ["ok", "warning", "critical"],
     "groups": [],
     "name": "FoundationDB Check",
-    "description": "Returns `CRITICAL` if the check can't access the page, `WARNING` if the search string was not found, or `OK` otherwise."
+    "description": "Returns critical if there's an error querying the json status, warning if there are degraded processes, or ok otherwise."
```

Comment:
*JSON :P Although I wonder if that's really relevant and we should just say "querying the cluster status" or some such